_posts/2019-08-15-height-56.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: post
+title: "Block Height 56 - It's private, part 2"
+categories:
+tags: []
+image: blockheight-56.jpg
+description: Waddle you think of my collection Blue?
```

Comment:
"Waddle" I thought...that a typo... but.... alas.... no

Reply:
Just a _really_ bad duck pun, I know... 😳